tests/v0/map_spec.rb

```diff
@@ -1,7 +1,7 @@
 require_relative '../spec_helper'
 
-describe 'Map Endpoints v1' do
-  url = 'v1/map'
+describe 'Map Endpoints v0' do
+  url = 'v0/map'
```

Comment:
good fix - I see for map spec v1, you add a leading `/`, can you do that for this/others to be consistent?

Reply:
yes